diffpy/snmf/tests/test_optimization.py

```diff
@@ -0,0 +1,19 @@
+import pytest
```

Comment:
I think the name of this test file also needs to be changed

Reply:
I will fix this